site/en/guide/intro_to_graphs.ipynb

```diff
@@ -606,7 +606,7 @@
         "id": "PUR7qC_bquCn"
       },
       "source": [
-        "`print` is a *Python side effect*, and there are [other differences](function#limitations) that you should be aware of when converting a function into a `Function`."
+        "`print` is a *Python side effect*, and there are [other differences](./function.ipynb#limitations) that you should be aware of when converting a function into a `Function`."
```

Comment:
Thanks @lucasSaavedra123 and @MarkDaoust. Section titles may change in future, unlike the file names, so let's use the following:

```suggestion
        "`print` is a *Python side effect*, and there are other differences that you should be aware of when converting a function into a `Function`. Learn more in the _Limitations_ section of the [Better performance with tf.function](function.ipynb) guide."
```